React/Views/RCTWrapperViewController.m

```diff
@@ -73,7 +73,25 @@ - (void)viewWillAppear:(BOOL)animated
     }
 
     UINavigationBar *bar = self.navigationController.navigationBar;
-    bar.barTintColor = _navItem.barTintColor;
+    float red, green, blue, alpha;
+    BOOL colorConversionSuccessful = [_navItem.barTintColor getRed:&red green:&green blue:&blue alpha:&alpha];
+    if(colorConversionSuccessful && alpha < 1.0f) {
+      CGRect rect = CGRectMake(0, 0, 1, 1);
+      UIGraphicsBeginImageContextWithOptions(rect.size, NO, 0);
+      [_navItem.barTintColor setFill];
+      UIRectFill(rect);
+      
+      UIImage *backgroundImage = UIGraphicsGetImageFromCurrentImageContext();
+      UIGraphicsEndImageContext();
+      
+      [bar setBackgroundImage:backgroundImage forBarMetrics:UIBarMetricsDefault];
+      bar.shadowImage = [UIImage new];
+      bar.translucent = YES;
```

Comment:
Is this needed? I am wondering because there is a little extra complexity needed to handle the case where a bar switches between a bar tint color with alpha and without alpha (bar.translucent should revert to its default state when going back to a bar tint color without alpha)